lib/appliance_console/external_httpd_configuration.rb

```diff
@@ -18,8 +19,8 @@ module ExternalHttpdConfiguration
       HTTP_KEYTAB         = "/etc/http.keytab"
 
       EXTERNAL_AUTH_FILE  = "conf.d/cfme-external-auth"
-      HTTPD_EXTERNAL_AUTH = "/etc/httpd/#{EXTERNAL_AUTH_FILE}"
-      HTTPD_CONFIG        = "/etc/httpd/conf.d/cfme-https-application.conf"
+      HTTPD_EXTERNAL_AUTH = Pathname.new(MiqApache.root_dir).join("etc/httpd/#{EXTERNAL_AUTH_FILE}")
+      HTTPD_CONFIG        = Pathname.new(MiqApache.root_dir).join("etc/httpd/conf.d/cfme-https-application.conf")
```

Comment:
@brandondunne @Fryguy @kbrock @jrafanie 
There is also the ambiguity of which conf directory? /etc/httpd/conf or /etc/httpd/conf.d